pkgs/development/php-packages/composer/default.nix

```diff
@@ -1,16 +1,17 @@
-{ lib, callPackage, fetchFromGitHub, php, unzip, _7zz, xz, git, curl, cacert, makeBinaryWrapper }:
+{ lib, callPackage, fetchgit, php, unzip, _7zz, xz, git, curl, cacert, makeBinaryWrapper }:
 
 php.buildComposerProject (finalAttrs: {
   composer = callPackage ../../../build-support/php/pkgs/composer-phar.nix { };
 
   pname = "composer";
   version = "2.6.4";
 
-  src = fetchFromGitHub {
-    owner = "composer";
-    repo = "composer";
+  # Using `fetchgit` prevent the `composer.lock` file from being stripped out
```

Comment:
s/prevent/prevents/g I think?

Also, not sure if I follow here. What exactly does that mean?

Reply:
I've updated the message, hope it's clearer now.

I also hope we can merge this PR quickly, if possible today.